packages/algob/src/internal/core/config/config-validation.ts

```diff
@@ -155,26 +99,30 @@ export function getValidationErrors(config: any): CfgErrors {  // eslint-disable
         errors.push(net, "token", token, "string");
       }
 
-      const netConfigResult = HttpNetworkType.decode(hcfg);
-      if (isLeft(netConfigResult)) {
-        errors.push(net, "", hcfg, "HttpNetworkConfig");
+      try {
+        HttpNetworkType.parse(hcfg);
+      } 
+      catch(e) {
+        if (e instanceof z.ZodError) {
+          console.log(parseZodError(e));
```

Comment:
please use `console.error` here

Reply:
ok , this line is not needed , deleting it